src/scss/common.scss

```diff
@@ -3,23 +3,23 @@
 
 $fontfamily: $font-family;  
 
-@mixin message-bubble($color, $textColor, $borderRadius: 15px) {
+@mixin message-bubble($color, $textColor, $borderRadius: 20px) {
   background-color: $color;
   color: $textColor;
   border-radius: $borderRadius;
-  padding: 11px 15px;
-  max-width: 215px;
+  padding: 9px 14px;
+  max-width: 222px;
```

Comment:
Should/Can we have some sizes as variables as well?

Reply:
Good question.
We can put this into a variable. There is a separate scss file just for variables.
For now I focused on getting the stylings applied.

I talked with Ofri and the customisation will focus for now only on the chat logo and the theme color.
But, this is something we could do if needed.